wordpress-mobile/WordPress-Android
# View all Clicks screen

This PR adds support for the new View all Clicks screen.

To test:
1. Go to Stats
2. Tap on the View more button under the Clicks card
3. Notice the new View all Clicks screen opens up
